static/js/components/Logo.jsx

```diff
@@ -1,13 +1,34 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-import styles from './Logo.css';
+import styled, { keyframes } from 'styled-components';
+
+
+const rotate = keyframes`
+  0% {
+    transform: rotate(60deg);
+  }
+  50% {
+    transform: rotate(-10deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+`;
+
+const StyledLogo = styled.img`
+  vertical-align: middle;
+  height: 50px;
+  margin-top: -0.5em;
+  animation-name: ${props => props.rotate ? rotate : null};
+  animation-duration: ${props => props.rotate ? '4s' : null};
```

Comment:
This seems to work as expected (when `props.rotate` is undefined, these rules don't render). This is probably some kind of styled-components anti-pattern though.